Content.Shared/Shuttles/BUIStates/ShuttleConsoleBoundInterfaceState.cs

```diff
@@ -20,17 +20,21 @@ public sealed class ShuttleConsoleBoundInterfaceState : RadarConsoleBoundInterfa
 
     public List<(NetEntity Entity, string Destination, bool Enabled)> Destinations;
 
+    public float ThrustLimit;
+
     public ShuttleConsoleBoundInterfaceState(
         FTLState ftlState,
         TimeSpan ftlTime,
         List<(NetEntity Entity, string Destination, bool Enabled)> destinations,
         float maxRange,
+        float thrustLimit,
         NetCoordinates? coordinates,
         Angle? angle,
         List<DockingInterfaceState> docks) : base(maxRange, coordinates, angle, docks)
     {
         FTLState = ftlState;
         FTLTime = ftlTime;
         Destinations = destinations;
+        ThrustLimit = thrustLimit;
     }
```

Comment:
Considering how large these states are you're better off just networking shuttleconsolecomponent instead, plus it will also let you use predicted messages in the UI instead.